This patch is needed to get autofs to build on later 2.3-series kernels, and
presumably will be needed for 2.4, at least until autofs 4.0.0 is finished.
--- autofs-3.1.4/daemon/automount.c.2.4	Wed Apr  5 15:49:18 2000
+++ autofs-3.1.4/daemon/automount.c	Wed Apr  5 15:49:55 2000
@@ -35,6 +35,8 @@
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <sys/time.h>
+/* Prevent inclusion of linux/string.h if we have one.  Aaaaaargh! */
+#define _LINUX_STRING_H_
 #include <linux/auto_fs.h>
 
 #include "automount.h"
